### ISSUE TITLE
Error"Framework PyTorch with communicator None is not supported on BatchAI." 

### ISSUE THREAD

**Issue by thomasshupeng** [Azure/MachineLearningNotebooks#79]
Hi,

I'm trying to  use HyperDrive run to do the hyperparameter sweep work for Faster-RCNN model. I'm using CNTK instead of PyTorch model.  

My customized container works fine when I submit a single experiment on BatchAI compute target,  but when I wrapped it into a HyperDrive run, I got following error message.  

Not sure how PyTorch came into the picture, I don't use it in my script.

Thanks
Shu

"<START>[2018-11-12T22:47:41.449665][API][INFO]Experiment created<END>\n""<START>[2018-11-12T22:47:41.671104][GENERATOR][INFO]Trying to sample '1' jobs from the hyperparameter space<END>\n""<START>[2018-11-12T22:47:41.757886][GENERATOR][INFO]Successfully sampled '1' jobs, they will soon be submitted to the execution target.<END>\n"<START>[2018-11-12T22:47:54.9298488Z][SCHEDULER][ERROR]Exception occurred while preparing the environment. Message='{"Error":{"Code":"UserError","Message":"**Framework PyTorch with communicator None is not supported on BatchAI."**,"Target":null,"Details":[],"InnerError":null,"DebugInfo":{"Type":"Microsoft.MachineLearning.Common.WebApi.Exceptions.BadRequestException","Message":"Framework PyTorch with communicator None is not supported on BatchAI.","StackTrace":"...skipped...","InnerException":null,"Data":{},"ErrorResponse":null}},"Correlation":{"operation":"f87346ff-464908009096e2d8","request":"bXr9tPY0hJE="}}'. InnerException=''. Stack trace:'   at Microsoft.MachineLearning.Execution.Services.BatchAiStrategy.CreateUnderlyingJob(StrategyState initialState, DynamicWorkingRunState workingState)...."

**Comment by mx-iao** [comment]
Hi @thomasshupeng, can you share the code you used to instantiate the estimator?

**Comment by thomasshupeng** [comment]
Sure. I'm using a customized container, but there is nothing special in it. 

script_params = {
    '--data_folder': ds.as_download(),
    '--local_run': False
}

...
computer_target = ws.compute_targets()
...

from azureml.train.estimator import Estimator
custom_docker_base_image = 'shpengws2740158968.azurecr.io/azureml/miniconda_py36_cntk-gpu_v4'
est = Estimator(source_directory=script_folder,
                script_params=script_params,
                compute_target=compute_target,
                distributed_backend='mpi',
                entry_script='train.py',
                pip_packages=['cntk-gpu==2.6', 'easydict','cython', 'opencv-python'],
                conda_packages = ['python==3.6','Scipy', 'pillow'],
                custom_docker_base_image=custom_docker_base_image,
                use_gpu=True
                )
